client/pd_service_discovery.go

```diff
@@ -732,6 +792,21 @@ func (c *pdServiceDiscovery) GetServiceClient() ServiceClient {
 	return leaderClient
 }
 
+// GetAllServiceClients implments ServiceDiscovery
+func (c *pdServiceDiscovery) GetAllServiceClients() []ServiceClient {
+	ret := make([]ServiceClient, 0)
```

Comment:
Is it better to specify `make(client, 1)` or another larger value? perhaps the number of clients in the main scenario is greater than 0.